### PR TITLE
Update snapshots-troubleshooting-backup-restore.md

### PR DESCRIPTION
**Pull request by nvanthao**
Update snapshot troubleshooting doc with common Velero issue

```
Warning: at least one source file could not be read
```

### CODE REVIEW

**Comment by nvanthao** [comment]
Many thanks @cbodonnell!

**Review by cbodonnell — APPROVED** [review]

File: docs/enterprise/snapshots-troubleshooting-backup-restore.md [hunk 1]
```diff
@@ -142,6 +142,28 @@ For more information, see the [Restic backup — OOM-killed on raspberry pi afte
...
+#### Cause
+
+There are file changes between Restic's initial scan of the volume and during the backup to Restic store.
+
+#### Solution
+
+To resolve this issue:
+
+1. Use [Snapshot Hooks](https://docs.replicated.com/vendor/snapshots-hooks) to export data to an [EmptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) volume and include that in the backup instead of the primary PVC volume.
+2. Freeze the file system to ensure all pending disk I/O operations have completed prior to taking a snapshot. For more information, see [Hook Example with fsfreeze](https://velero.io/docs/main/backup-hooks/#hook-example-with-fsfreeze) in the Velero documentation.
```
Comment (paigecalvert) [inline docs/enterprise/snapshots-troubleshooting-backup-restore.md:164]: @nvanthao or @cbodonnell , just to confirm: is this supposed to be a procedure (as in, you do step #1 and then do step #2), or are these two different options to resolve the issue (as is, you can do one or the other)?

**Review by paigecalvert — DISMISSED** [review]
Thanks for adding this in @nvanthao ! I added just one comment with a question.

File: docs/enterprise/snapshots-troubleshooting-backup-restore.md (hunk 1 above)
Comment (cbodonnell) [reply docs/enterprise/snapshots-troubleshooting-backup-restore.md:164]: @paigecalvert it's the latter, you can do one or the other to get around this issue

**Review by paigecalvert — APPROVED** [review]

**Pull request merged**